### DISCUSSION TITLE
layouts.css

### DISCUSSION THREAD

**Discussion by argyleink** [argyleink/open-props#69] [Ideas]
some explorations here https://github.com/argyleink/open-props/blob/main/src/props.layouts.css

the mixin route is interesting, but could be a totally separate library idea. also means shipping the props on `*` instead of `html`.

**Comment by roblevintennis** [comment]
I didn't notice this until after commenting on the related discussion Q&A I started. So it looks like this IS something you're contemplating in terms of if/how it should belong in OP. I think its pretty cool if it has examples and some guidance but then again I still don't really know how to set it up so I'm not a good judge of its merit. It certainly "tripped me up" a bit as you know from my other feedback 😋
